### PR TITLE
Add line-height to tips to fix appearance on line-breaks

### PR DESCRIPTION
**Pull request by michaelgmcd**

### CODE REVIEW

File: static/messages.less [hunk 1]
```diff
@@ -18,6 +18,7 @@ ul.background-message {
   padding: 0;
 
   li {
+    line-height: 42px;
```
Comment (thomasjo) [inline static/messages.less]: We should use some manner of relative size here. Pretty sure this is well-behaved for non-default font sizes. /cc @simurai. 

---
_The choice of 42 is... "cute"_ 😁

Comment (mgmcdermott) [inline static/messages.less]: Yes. That makes a lot of sense. I'll update accordingly.

**Comment by mgmcdermott** [comment]
@thomasjo I've updated to use `font-size * 4`

**Comment by simurai** [comment]
@mgmcdermott do you mind posting a screenshot? Currently the `line-height` gets inherited from `body` which is `1.42857143`. So it should look ok for multi-line tips:

screen shot 2016-06-21 at 11 05 35 am

Maybe you have a theme installed that overrides it?

**Comment by mgmcdermott** [comment]
screen shot 2016-06-20 at 10 31 04 pm

**Comment by simurai** [comment]
Thanks, still can't reproduce it with those two themes installed.

Maybe another package? How does it look when you start Atom with `atom . --safe`. Then only bundled packages will be loaded.

Also, are you using Atom stable? `1.8`?

**Comment by simurai** [comment]
@mgmcdermott Or in case you're familiar with the DevTools, do you see some `line-height` applied when inspecting:

screen shot 2016-06-21 at 1 09 27 pm

**Comment by mgmcdermott** [comment]
Sorry for the lack of info here. It looks like trying to set the editor line-height also set the line-height of these messages. My fault.

**Pull request closed without merging**